src/notation/qml/MuseScore/NotationScene/internal/TextStylePopup.qml

```diff
@@ -0,0 +1,389 @@
+                        },
+                        {
+                            iconRole: IconCode.TEXT_ALIGN_BASELINE,
+                            typeRole: TextTypes.FONT_ALIGN_V_BASELINE,
+                            title: qsTrc("inspector", "Align baseline"),
+                            description: qsTrc("inspector", "Align baseline of text to reference point")
+                        }
+                    ]
+
+                    delegate: FlatRadioButton {
+                        toolTipTitle: modelData.title
+                        toolTipDescription: modelData.description
+
+                        width: 30
+                        transparent: true
+
+                        iconCode: modelData.iconRole
+                        checked: root.model && !root.model.verticalAlignment.isUndefined && (root.model.verticalAlignment.value === modelData.typeRole)
+                        onToggled: {
+                            root.model.verticalAlignment.value = modelData.typeRole
+                        }
+                    }
+                }
+            }
+
+            //Column -> 2 | Row -> 2
+            RowLayout {
+
+                //Text subScript and superScript buttons.
+                RadioButtonGroup {
+                    id: subscriptOptionsButtonList
+                    height: 30
+
+                    model: [
+                        { iconRole: IconCode.TEXT_SUBSCRIPT, typeRole: TextTypes.TEXT_SUBSCRIPT_BOTTOM, titleRole: qsTrc("inspector", "Subscript") },
+                        { iconRole: IconCode.TEXT_SUPERSCRIPT, typeRole: TextTypes.TEXT_SUBSCRIPT_TOP, titleRole: qsTrc("inspector", "Superscript") }
+                    ]
+
+                    delegate: FlatRadioButton {
+
+                        width: 30
+                        transparent: true
+
+                        iconCode: modelData["iconRole"]
+                        checked: root.model && !root.model.textScriptAlignment.isUndefined ? root.model.textScriptAlignment.value === modelData["typeRole"]
+                                                                                           : false
+                        onClicked: {
+                            if (root.model.textScriptAlignment.value === modelData["typeRole"]) {
+                                root.model.textScriptAlignment.value = TextTypes.TEXT_SUBSCRIPT_NORMAL
+                            } else {
+                                root.model.textScriptAlignment.value = modelData["typeRole"]
+                            }
+                        }
+                    }
+                }
+
+                //Line Spacing Label
+                Text {
+                    id: lineSpacingLabel
+
+                    readonly property bool isEmpty: text.length === 0
+
+                    color: ui.theme.fontPrimaryColor
+                    linkColor: ui.theme.linkColor
+                    opacity: 1.0
+
+                    text: 'Line Spacing:'
```

Comment:
Needs to be translatable, and "Sentence case" instead of "Title Case":
```suggestion
                    text: qsTrc("inspector", "Line spacing:")
```